backend/resources/medshare-data-sample-reformat.csv

```diff
@@ -1,5 +1,5 @@
 ProductRef,ProductName,ProductCatID,Category Name,Sub Category,UnitMeasureName
-700001,Misc. Medical Supplies and Equipment,1,Medical Equipment,Unsorted,LB
+700001,"Wrench, Open End, 3 1/2""",1,Medical Equipment,Unsorted,LB
```

Comment:
How did this change come about?

Reply:
My bad. Was playing around with the data set